scripts/dream.py

```diff
@@ -21,337 +13,23 @@
 
 def main():
     """Initialize command-line parsers and the diffusion model"""
-    opt  = Args()
-    args = opt.parse_args()
+    arg_parser  = Args()
+    args = arg_parser.parse_args()
     if not args:
         sys.exit(-1)
-
-    if args.laion400m:
-        print('--laion400m flag has been deprecated. Please use --model laion400m instead.')
-        sys.exit(-1)
-    if args.weights:
-        print('--weights argument has been deprecated. Please edit ./configs/models.yaml, and select the weights using --model instead.')
-        sys.exit(-1)
-
-    print('* Initializing, be patient...\n')
-    sys.path.append('.')
-    from ldm.generate import Generate
-
-    # these two lines prevent a horrible warning message from appearing
-    # when the frozen CLIP tokenizer is imported
-    import transformers
-    transformers.logging.set_verbosity_error()
-
-    # creating a simple Generate object with a handful of
-    # defaults passed on the command line.
-    # additional parameters will be added (or overriden) during
-    # the user input loop
-    try:
-        gen = Generate(
-            conf           = opt.conf,
-            model          = opt.model,
-            sampler_name   = opt.sampler_name,
-            embedding_path = opt.embedding_path,
-            full_precision = opt.full_precision,
-        )
-    except (FileNotFoundError, IOError, KeyError) as e:
-        print(f'{e}. Aborting.')
-        sys.exit(-1)
-
-    # make sure the output directory exists
-    if not os.path.exists(opt.outdir):
-        os.makedirs(opt.outdir)
-
-    # load the infile as a list of lines
-    infile = None
-    if opt.infile:
-        try:
-            if os.path.isfile(opt.infile):
-                infile = open(opt.infile, 'r', encoding='utf-8')
-            elif opt.infile == '-':  # stdin
-                infile = sys.stdin
-            else:
-                raise FileNotFoundError(f'{opt.infile} not found.')
-        except (FileNotFoundError, IOError) as e:
-            print(f'{e}. Aborting.')
-            sys.exit(-1)
-
-    if opt.seamless:
-        print(">> changed to seamless tiling mode")
-
-    # preload the model
-    gen.load_model()
-
-    if not infile:
-        print(
-            "\n* Initialization done! Awaiting your command (-h for help, 'q' to quit)"
-        )
-
-    # web server loops forever
-    if opt.web:
-        dream_server_loop(gen, opt.host, opt.port, opt.outdir)
+    
+    if args.web:
+      # Start server
+      try:
+        print('The --web flag is being deprecated. Please run server/application.py instead')
```

Comment:
I removed the comment. I wasn't sure if others would prefer to just run the scripts for console/web app directly, rather than this thin script that just calls into each. Better to not break anyone though.